### PR TITLE
set up skeleton for dwds

### PR DESCRIPTION
**Pull request by jakemac53**
Just gets things started, no actual impl here yet

I added just a couple deps which I know we will have to use eventually, can remove them if we want.

I also added a skeleton for the binary itself and a test, so that I could set up travis.

### CODE REVIEW

File: dwds/pubspec.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+dependencies:
+  source_maps: ^0.10.0
+  vm_service_lib: ^3.1.5-dev
+  webkit_inspection_protocol: ^0.3.6
+dev_dependencies:
+  test: ^1.0.0
+dependency_overrides:
+  vm_service_lib:
+    git:
+      url: https://github.com/jakemac53/vm_service_drivers.git
```
Comment (grouma) [inline dwds/pubspec.yaml]: Why this?

File: dwds/pubspec.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+name: dwds
+version: 0.1.0-dev
+description: >-
+  A service that proxies between the Chrome debug protocol and the vm
+  service protocol.
+environment:
+  sdk: ^2.1.0
+dependencies:
+  source_maps: ^0.10.0
```
Comment (grouma) [inline dwds/pubspec.yaml]: Should we keep this empty until we actually need them?

File: dwds/pubspec.yaml (hunk 1 above)
Comment (jakemac53) [reply dwds/pubspec.yaml]: Just until the PR for it lands https://github.com/dart-lang/vm_service_drivers/pull/213

File: dwds/pubspec.yaml (hunk 2 above)
Comment (jakemac53) [reply dwds/pubspec.yaml]: We can but I can guarantee you its necessary so I was just trying to reduce merge conflicts

**Pull request merged**